### PR TITLE
update init script

### PR DESCRIPTION
**Pull request by fedekunze**
## Description

update `init.sh` script to enable faucet

______

For contributor use:

- [ ] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#pr-targeting))
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/master/docs/building-modules/structure.md).
- [ ] Wrote unit and integration [tests](https://github.com/cosmos/cosmos-sdk/blob/master/CONTRIBUTING.md#testing)
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [ ] Added  relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [ ] Re-reviewed `Files changed` in the Github PR explorer

______

For admin use:

- [ ] Added appropriate labels to PR (ex. `WIP`, `R4R`, `docs`, etc)
- [ ] Reviewers assigned
- [ ] Squashed all commits, uses message "Merge pull request #XYZ: [title]" ([coding standards](https://github.com/tendermint/coding/blob/master/README.md#merging-a-pr))

### CODE REVIEW

File: init.sh [hunk 1]
```diff
@@ -32,13 +32,21 @@ emintd gentx --name $KEY --keyring-backend test
 # Collect genesis tx
 emintd collect-gentxs
 
+# Enable faucet
+cat  $HOME/.emintd/config/genesis.json | jq '.app_state["faucet"]["enable_faucet"]=true' >  $HOME/.emintd/config/tmp_genesis.json && mv $HOME/.emintd/config/tmp_genesis.json $HOME/.emintd/config/genesis.json
+
+echo -e '\n\ntestnet faucet enabled'
+echo -e 'to transfer tokens to your account address use:'
+echo -e "emintcli tx faucet fund 100photon --from $KEY\n"
```
Comment (noot) [inline init.sh]: does this transfer tokens to your account, or from your account to the faucet?

**Review by noot — APPROVED** [review]
looks good, just have a question about what the fund command does

File: init.sh (hunk 1 above)
Comment (fedekunze) [reply init.sh]: it mints new tokens and transfers them to your account, updating the total supply accordingly

Comment (fedekunze) [reply init.sh]: do you want me to rename the command from `fund` to `request`?

Comment (noot) [reply init.sh]: ah okay, yeah that might make more sense. or, you could rename `from` to `to`
but renaming the command to `request` might make more sense, I'm good with either

Comment (fedekunze) [reply init.sh]: renamed

Comment (noot) [reply init.sh]: looks good! tested and works

**Pull request merged**